parsl/monitoring/visualization/app.py

```diff
@@ -23,6 +23,7 @@ def cli_run():
     app = Flask(__name__)
     app.config['SQLALCHEMY_DATABASE_URI'] = args.db_path
     app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    app.config['SECRET_KEY'] = os.urandom(24)
```

Comment:
this is not related to provenance, i think - if it's something useful for Parsl in general, make a separate PR

Reply:
Actually, it is required for the provenance. It is required in order to use the file query form in parsl-visualizer